firstQuestion.js

```diff
@@ -0,0 +1,43 @@
+const set1= new Array(3,1,7,9);
```

Comment:
No need here to use the `Array` constructor function, simply use the literal notation:

```js
const set1 = [3, 1, 7, 9];
```

Also there's no need to submit your solution with the values you tested with, only the solution itself is needed.